app/src/main-process/main.ts

```diff
@@ -276,6 +276,11 @@ app.on('ready', () => {
     reportError(error, extra)
   })
 
+  ipcMain.on('open-external', (event: Electron.IpcMessageEvent, { id, path }: { id: number, path: string }) => {
```

Comment:
I thought I'd need to track the IPC request but that's unnecessary :fire::fire::fire: